subsys/bluetooth/Kconfig

```
@@ -115,14 +115,20 @@ config BT_CONN_TX
 	help
 	  Hidden configuration that is true if ACL or broadcast ISO is enabled
 
+config BT_LE_LOCAL_MINIMUM_REQUIRED_FEATURE_PAGE
+	int
+	default 1 if BT_FRAME_SPACE_UPDATE
```

Should there be a `default 0` as a fallback here incase the application for whatever reason enables `BT_LE_EXTENDED_FEAT_SET` but none of the features that requires it?

```suggestion
	default 1 if BT_FRAME_SPACE_UPDATE
	default 0
```

---

Great point